cmake/tribits/doc/developers_guide/TribitsDevelopersGuide.rst

```diff
@@ -6581,8 +6616,8 @@ One can also change what compilers are written into the generated
 ``<Project>Config.cmake`` and ``<Package>Config.cmake`` files for the build
 and the install trees.  By default, the compilers pointed to in these
 ``Config.cmake`` files will be ``CMAKE_<LANG>_COMPILER`` where ``<LANG>`` =
-``CXX``, ``C``, and ``Fortran``.  But if can change this by setting any of the
-following::
+``CXX``, ``C``, and ``Fortran``.  But if one can change this by setting any of
```

Comment:
```suggestion
``CXX``, ``C``, and ``Fortran``, but one can change this by setting any of
```

Reply:
Addressed in commit https://github.com/TriBITSPub/TriBITS/commit/c7cc94989ba586612113da5e266ec1201ac63191.